emspy/query/fltquery.py

```diff
@@ -449,13 +457,30 @@ def load_metadata(self, file_name = None):
 ## Experimental...
 
 basic_ops = {
-	'==': 'equal', '!=': 'notEqual', '<': 'lessThan',
-	'<=': 'lessThanOrEqual', '>': 'greaterThan',
+	'==': 'equal',
+	'!=': 'notEqual',
+	'<': 'lessThan',
+	'<=': 'lessThanOrEqual',
+	'>': 'greaterThan',
 	'>=': 'greaterThanOrEqual'
 }
+between_ops = {
+	'<': {
```

Comment:
Have you made sure this works for dates?  I don't think dates support one of these through the API (it's either <= or >=).

Reply:
I included between filters just for numeric fields, datetime fields would raise ValueError, I need to test this better

Reply:
I added test cases for this, checking that trying to filter a datetime on a range raises an error, although the dateTimeRange operator is supported by the API and could be included easily.